src/main/java/study/racing/ui/ResultView.java

```diff
@@ -1,26 +1,55 @@
 package study.racing.ui;
 
-import study.racing.model.Result;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import study.racing.model.Name;
 import study.racing.model.car.Car;
+import study.racing.model.result.GameResults;
+import study.racing.model.result.Round;
 
 public final class ResultView {
 
-    private static final String DISTANCE_SYMBOL = "-";
     private static final StringBuilder builder = new StringBuilder();
+    private static final String DISTANCE_SYMBOL = "-";
+    public static final String NAME_DISTANCE_DELIMITER = " : ";
+    public static final String WINNERS_DELIMITER = ", ";
+    public static final String WINNER_MESSAGE = "가 최종 우승했습니다.";
 
     private ResultView() {
     }
 
-    public static void printResult(Result result) {
-        result.getResult().forEach(car -> doPrint(car));
+    public static void printResult(GameResults gameResults) {
+        gameResults.allRoundResults()
+                   .forEach(ResultView::doPrint);
+        printWinner(gameResults);
+    }
+
+    private static void doPrint(Round round) {
+        round.result()
+             .getCars()
+             .forEach(ResultView::doPrint);
         System.out.println();
     }
 
     private static void doPrint(Car car) {
         builder.setLength(0);
-        for (int i = 0; i < car.getDistance(); i++) {
+        builder.append(car.getName() + NAME_DISTANCE_DELIMITER);
```

Comment:
`car.getName() + NAME_DISTANCE_DELIMITER`
이 코드도 사실 String 객체 두개를 합쳐서 새로운 String 객체를 만드는 것입니다. 그래서 다음과 같이 두 번의 append로 나누는게 더 나으실 거에요 👍 

```suggestion
       builder.append(car.getName())
                .append(NAME_DISTANCE_DELIMITER);
```